OData/test/E2ETest/WebStack.QA.Test.OData/Formatter/Untyped/UntypedDeltaSerializationTests.cs

```diff
@@ -0,0 +1,154 @@
+﻿using System;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Web.Http;
+using System.Web.OData;
+using System.Web.OData.Builder;
+using System.Web.OData.Extensions;
+using System.Web.OData.Routing;
+using System.Web.OData.Routing.Conventions;
+using Microsoft.OData.Core;
+using Microsoft.OData.Edm;
+using Microsoft.OData.Edm.Library;
+using Nuwa;
+using WebStack.QA.Test.OData.Common;
+using Xunit;
+using Xunit.Extensions;
+
+namespace WebStack.QA.Test.OData.Formatter.Untyped
+{
+    [NuwaFramework]
+    [NuwaHttpClientConfiguration(MessageLog = false)]
+    [NuwaTrace(typeof(PlaceholderTraceWriter))]
+    public class UntypedDeltaSerializationTests
+    {
+        private string baseAddress = null;
+
+        [NuwaBaseAddress]
+        public string BaseAddress
+        {
+            get
+            {
+                return baseAddress;
+            }
+            set
+            {
+                if (!string.IsNullOrEmpty(value))
+                {
+                    this.baseAddress = value.Replace("localhost", Environment.MachineName);
+                }
+            }
+        }
+
+        [NuwaHttpClient]
+        public HttpClient Client { get; set; }
+
+        [NuwaConfiguration]
+        public static void UpdateConfiguration(HttpConfiguration configuration)
+        {
+            configuration.IncludeErrorDetailPolicy = IncludeErrorDetailPolicy.Always;
+            configuration.Formatters.JsonFormatter.SerializerSettings.ReferenceLoopHandling = Newtonsoft.Json.ReferenceLoopHandling.Ignore;
+            configuration.MapODataServiceRoute("untyped", "untyped", GetEdmModel(), new DefaultODataPathHandler(), ODataRoutingConventions.CreateDefault());
+        }
+
+        private static IEdmModel GetEdmModel()
+        {
+            ODataModelBuilder builder = new ODataConventionModelBuilder();
+            var customers = builder.EntitySet<UntypedCustomer>("UntypedDeltaCustomers");
+            customers.EntityType.Property(c => c.Name).IsRequired();
+            var orders = builder.EntitySet<UntypedOrder>("UntypedDeltaOrders");
+            return builder.GetEdmModel();
+        }
+
+        [Theory]
+        [InlineData("application/json")]
+        [InlineData("application/json;odata.metadata=minimal")]
+        [InlineData("application/json;odata.metadata=full")]
+        public void UntypedDeltaWorksInAllFormats(string acceptHeader)
```

Comment:
As discussed with you, handling of Delta Token is the responsibility of the Server, so could not have a negative scenario here